api/app/reviews/tests.py

```diff
@@ -1285,3 +1288,200 @@ def test_total_number_of_pages_when_zero(self):
+                     True, None, options)
+            Question(1, sections[1].id,
+                     'title', 'Enter 50 to 150 words', 1, 'long_text', ''),
+            Question(1, sections[1].id,
+                     'firstname', 'Enter 50 to 150 words', 2, 'long_text', ''),
+            Question(1, sections[1].id,
+                     'lastname', 'Enter 50 to 150 words', 3, 'long_text', ''),
+            Question(1, sections[1].id,
+                     'email', 'Enter 50 to 150 words', 4, 'long_text', ''),
+        ]
+
+        questions[5].key = 'nomination_title'
+        questions[6].key = 'nomination_firstname'
+        questions[7].key = 'nomination_lastname'
+        questions[8].key = 'nomination_email'
+        db.session.add_all(questions)
+        db.session.commit()
+
+
+        closed_review = ReviewForm(2, datetime(2018, 4, 30))
+        closed_review.close()
+        review_forms = [
+            ReviewForm(1, datetime(2019, 4, 30)),
+            closed_review
+        ]
+        db.session.add_all(review_forms)
+        db.session.commit()
+
+        review_configs = [
+            ReviewConfiguration(review_form_id=review_forms[0].id, num_reviews_required=3, num_optional_reviews=0),
+            ReviewConfiguration(review_form_id=review_forms[1].id, num_reviews_required=3, num_optional_reviews=0)
+        ]
+        db.session.add_all(review_configs)
+        db.session.commit()
+
+        review_questions = [
+            ReviewQuestion(1, 1, None, None, 'multi-choice', None, None, True, 1, None, None, 0),
+            ReviewQuestion(1, 2, None, None, 'multi-choice', None, None, True, 2, None, None, 0),
+            ReviewQuestion(2, 3, None, None, 'multi-choice', None, None, True, 1, None, None, 0),
+            ReviewQuestion(2, 4, None, None, 'information', None, None, False, 2, None, None, 0)
+        ]
+        db.session.add_all(review_questions)
+        db.session.commit()
+
+        self.test_response = Response(  # Nominating other
+            self.application_form.id, self.first_user.id)
+
+        self.add_to_db(self.test_response)
+        answers = [
+            Answer(self.test_response.id, questions[5].id, 'Mx'),
+            Answer(self.test_response.id, questions[6].id, 'Skittles'),
+            Answer(self.test_response.id, questions[7].id, 'Cat'),
+            Answer(self.test_response.id, questions[8].id, 'skittles@box.com'),
+        ]
+        db.session.add_all(answers)
+        db.session.commit()
+
+        self.first_headers = self.get_auth_header_for("firstuser@mail.com")
+
+        db.session.flush()
+
+    def test_get_reference_request_by_event_id(self):
+        self.seed_static_data()
+        reference_req = ReferenceRequest(1, 'Mr', 'John', 'Snow', 'Supervisor', 'common@email.com')
+        reference_request_repository.create(reference_req)
+        REFERENCE_DETAIL = {

```

Comment:
You still need to add the reference to the system, like you do in the next test with the call to `post`